### DISCUSSION TITLE
How to export the cvxpy problem into a numerical solver such as gurobi/mosek data file?

### DISCUSSION THREAD

**Discussion by JiaxinWang-thu** [cvxpy/cvxpy#1715] [Q&A]
As is known, the Gurobi/Mosek solver can export a standard model file containing coefficiency matrix and so on. But I cannot find some API to do the same thing using cvxpy, could someone please help me?

**Comment by rileyjmurray** [comment]
For mosek, you can do ``prob.solve(solver='MOSEK', save_file='file_name_here.tsk')``. So far, no one has implemented similar functionality for our Gurobi interface.

You can find more details on solver options here: https://www.cvxpy.org/tutorial/advanced/index.html#setting-solver-options.
